tests/Event/Http/CommonHttpTest.php

```diff
@@ -100,13 +100,14 @@ public function test request with multivalues query string have basic su
 
         $this->assertQueryParameters([
             'foo' => ['bar', 'baz'],
+            'date' => ['2021-07-06', '2021-08-06'],
             'cards' => ['birthday'],
             'colors' => [['red'], ['blue']],
             'shapes' => ['a' => ['square', 'triangle']],
             'myvar' => 'abc',
         ]);
-        $this->assertQueryString('foo%5B0%5D=bar&foo%5B1%5D=baz&cards%5B0%5D=birthday&colors%5B0%5D%5B0%5D=red&colors%5B1%5D%5B0%5D=blue&shapes%5Ba%5D%5B0%5D=square&shapes%5Ba%5D%5B1%5D=triangle&myvar=abc');
-        $this->assertUri('/path?foo%5B0%5D=bar&foo%5B1%5D=baz&cards%5B0%5D=birthday&colors%5B0%5D%5B0%5D=red&colors%5B1%5D%5B0%5D=blue&shapes%5Ba%5D%5B0%5D=square&shapes%5Ba%5D%5B1%5D=triangle&myvar=abc');
+        $this->assertQueryString('foo%5B0%5D=bar&foo%5B1%5D=baz&date%5B0%5D=2021-07-06&date%5B1%5D=2021-08-06&cards%5B0%5D=birthday&colors%5B0%5D%5B0%5D=red&colors%5B1%5D%5B0%5D=blue&shapes%5Ba%5D%5B0%5D=square&shapes%5Ba%5D%5B1%5D=triangle&myvar=abc');
```

Comment:
Are we expecing this to have the `data[0]` and `date[1]` as part of the url now?

Reply:
we send `date=xxx&date=yyy` and bref layer converts to `date[0]=xxx&date[1]=yyy` then send it down to laravel which will handle it converting back to `date=xxx&date=yyy`